components/ActionsList/index.js

```diff
@@ -9,40 +9,43 @@ import Table, {
   TableData
 } from 'components/Table';
 
-const ActionsList = ({ actions }) => (
-  <Table className="lbh-actions-list__table">
-    <TableHead>
-      <TableRow className="lbh-actions-list__header">
-        <TableHeader scope="col">Description</TableHeader>
-        <TableHeader scope="col" className="lbh-actions-list__due-date">
-          Due date
-        </TableHeader>
-      </TableRow>
-    </TableHead>
-    <TableBody>
-      {actions.map(action => (
-        <TableRow key={action.title}>
-          <TableData className="lbh-actions-list__description">
-            <Heading as="h2" size="m">
-              {action.title}
-            </Heading>
-            <ExpandingText
-              expandButtonText="Show details"
-              contractButtonText="Hide details"
-            >
-              {action.description}
-            </ExpandingText>
-            <div className="lbh-actions-list__descriptions-mobile">
-              Due <DueDate dateTime={action.dueDate} />
-            </div>
-          </TableData>
-          <TableData className="lbh-actions-list__due-date">
-            <DueDate dateTime={action.dueDate} />
-          </TableData>
+const ActionsList = ({ actions }) => {
+  if (!actions || actions.length === 0) return <div></div>;
```

Comment:
do we need an empty div here or can we just return the table if there are actions?

Reply:
I think render needs to return something, so I just went for a div. Maybe it's just worth doing the check in summary page and based on that add or not add the component?

Reply:
yeah, looks like tom has already done that in the other PR